lib/iob/history.js

```diff
@@ -366,18 +366,18 @@ function calcTempTreatments (inputs, zeroTempDuration) {
             tempHistory.push(temp);
         }
         // Add a temp basal cancel event to ignore future temps and reduce predBG oscillation
-        var temp = {};
-        temp.rate = 0;
-        // start the zero temp 1m in the future to avoid clock skew
-        temp.started_at = new Date(now.getTime() + (1 * 60 * 1000));
-        temp.date = temp.started_at.getTime();
-        if (zeroTempDuration) {
-            temp.duration = zeroTempDuration;
-        } else {
-            temp.duration = 0;
-        }
-        tempHistory.push(temp);
     }
+    var temp = {};
```

Comment:
this is a "cut-off" TBR that oref0 inserts at "now" timestamp
previously, it was inserted inside the loop - but it only needs to be inserted once